src/dso.scss

```diff
@@ -17,17 +17,19 @@ $library: true !default;
 @import "styles/mixins/bootstrap-variables";
 
 /*!
- * Bootstrap v3.3.7 (http://getbootstrap.com)
- * Copyright 2011-2016 Twitter, Inc.
- * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
- */
+* Bootstrap v3.3.7 (http://getbootstrap.com)
+* Copyright 2011-2016 Twitter, Inc.
+* Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+*/
```

Comment:
Whitespaces terugzetten, onnodige change. Daarbij is de conventie juist dat de sterretjes uitlijnen

Reply:
Geen idee wie of wat het heeft weggehaald. Zal VS code wel verzonnen hebben?